models/order/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@commercetools-test-data/order",
+  "version": "4.10.0",
+  "description": "Data model for commercetools API Order",
+  "bugs": "https://github.com/commercetools/test-data/issues",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/commercetools/test-data.git",
+    "directory": "models/order"
+  },
+  "keywords": ["javascript", "typescript", "test-data"],
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "dist/commercetools-test-data-order.cjs.js",
+  "module": "dist/commercetools-test-data-order.esm.js",
+  "files": ["dist", "package.json", "LICENSE", "README.md"],
+  "dependencies": {
+    "@babel/runtime": "^7.17.9",
+    "@babel/runtime-corejs3": "^7.17.9",
+    "@commercetools-test-data/commons": "4.10.0",
+    "@commercetools-test-data/core": "4.10.0",
+    "@commercetools-test-data/utils": "4.10.0",
+    "@commercetools-test-data/line-item": "4.10.0",
+    "@commercetools-test-data/cent-precision-money": "4.10.0",
+    "@commercetools-test-data/customer-group": "4.10.0",
+    "@commercetools-test-data/cart": "4.10.0",
+    "@commercetools-test-data/cart-discount": "4.10.0",
+    "@commercetools/platform-sdk": "^4.5.0",
```

Comment:
Does it matter that _just_ this module has the updated version? Do we need to update it across all modules?

Reply:
the current SDK we were using on all our modules (4.0.0 I think?) was missing some features/fields on the Order type. I had to update the package here to access the new fields. Basically wondering if we need to update the SDK across all models or if it's fine here just being more up-to-date

Reply:
Although shouldn't the `^` symbol download the most recent package within that semver anyway? Now I'm confused 🤔

Reply:
I've reverted this in 35c3527 which didn't seem to cause any issues? 🤔

Reply:
Ok, I think I just needed to nuke my `node_modules` folder and re-install to get the new version